Makefile

```diff
@@ -85,9 +85,12 @@ endif
 	mkdir -p build
 	touch $@
 
-oldtest: | nvim
+oldtest: | nvim tags
```

Comment:
I debated between this or generating minimal help and tags files in the test itself.  I had originally (when first picking up brcolow's work) done the latter but then revised the test to just set 'runtimepath'.